openssl/src/pkcs7.rs

```diff
@@ -281,11 +296,37 @@ impl Pkcs7Ref {
             Ok(stack)
         }
     }
+
+    // Return the type of a PKCS#7 structure as an Asn1Object
+    pub fn type_oid(&self) -> Option<&Asn1ObjectRef> {
```

Comment:
Should this just be called `type`?

Reply:
Can't use that here, unfortunately, because `type` is a [strict keyword][1].

[1]: https://doc.rust-lang.org/reference/keywords.html#strict-keywords

Reply:
It could be `r#type` instead to use the raw identifier syntax, but yeah -- I think bare `type` won't work.

Reply:
You can go with a raw ident, but `type_` is a bit easier to work with.

Reply:
Fixed (changed to `type_`)